### PR TITLE
Feat/angular material/unify text test

### PR DESCRIPTION
**Pull request by eneufeld**
based on #1092

### CODE REVIEW

File: packages/angular-test/src/text.ts [hunk 1]
```diff
@@ -10,11 +10,13 @@ export const textTest = <C extends JsonFormsControl>(
...
+    elementToUse: (element: DebugElement) => any) => () => {
         let fixture: ComponentFixture<any>;
-        let textNativeElement: HTMLInputElement;
-        let component: C;
         let textElement: DebugElement;
+        let textNativeElement;
+        let component: C;
 
         const data = { foo: 'foo' };
         const schema: JsonSchema = {
```
Comment (edgarmueller) [inline packages/angular-test/src/text.ts]: I missed this before, but on line 26 foo has a type of 'text', which should be 'string', I think. Which brings me to the question whether this `schema` has any actual effect on the tests. Can we clarify that?

File: packages/angular-test/src/text.ts [hunk 2]
```diff
@@ -250,7 +254,7 @@ export const textTest = <C extends JsonFormsControl>(
             mockSubStore.complete();
             fixture.detectChanges();
             component.ngOnInit();
-            expect(textNativeElement.id).toBe('myId');
+            expect(textElement.nativeElement.id).toBe('myId');
```
Comment (edgarmueller) [inline packages/angular-test/src/text.ts]: Shouldn't we use `textNativeElement` here?

File: packages/angular-test/src/text.ts [hunk 3]
```diff
@@ -10,11 +10,13 @@ export const textTest = <C extends JsonFormsControl>(
     imports: any[],
     providers: any[],
     componentUT: Type<C>,
-    errorTestInformation: ErrorTestInformation) => () => {
+    instance: string,
+    errorTestInformation: ErrorTestInformation,
+    elementToUse: (element: DebugElement) => any) => () => {
```
Comment (edgarmueller) [inline packages/angular-test/src/text.ts]: Minor: Use consistent naming for parameter `elementToUse` and arguments `toSelect`, maybe `nativeElementSelector` or similar.

File: packages/angular-test/src/text.ts (hunk 2 above)
Comment (eneufeld) [reply packages/angular-test/src/text.ts]: no as textNativeElement could be in a wrapper and we can't be sure that the propagation is working as expected, eg for angular-material your suggestions works, but for ionic it fails

File: packages/angular-test/src/text.ts (hunk 3 above)
Comment (eneufeld) [reply packages/angular-test/src/text.ts]: I would add this to #1100 and do a full clean up

File: packages/angular-test/src/text.ts (hunk 2 above)
Comment (edgarmueller) [reply packages/angular-test/src/text.ts]: We shoud improve naming then (maybe via #1100), I'd think that both `textNativeElement` and `textElement.nativeElement` are likely to be used as synonymous.

File: packages/angular-test/src/text.ts (hunk 1 above)
Comment (eneufeld) [reply packages/angular-test/src/text.ts]: should be fixed with 2 commit

**Pull request merged**